core/flyout.js

```diff
@@ -427,9 +581,14 @@ Blockly.Flyout.prototype.show = function(xmlList) {
   this.listeners_.push(Blockly.bindEvent_(this.svgBackground_, 'mouseover',
       this, deselectAll));
 
-  this.width_ = 0;
+  if (this.horizontalLayout_) {
```

Comment:
Argh. I wish this thing never collapsed function definitions.  It would be much easier for me to read if I could always glance up and see what function the diff was in.